### ISSUE TITLE
nothing should change size on hover

### ISSUE THREAD

**Issue by moates** [moates/onlyBans#30]
For the real-story pages...
- Possible thing to do instead: change the contrast of the photos to black and white
- Display both images side by side or the selected image and a box that the player could select to reveal the original image. Nothing should change via hover and no reveal should occur that does not require an action such as a link.

For photo selector....
- Possible thing to do instead: change the contrast of the photos to black
and white
